statement.md

```diff
@@ -0,0 +1,84 @@
+# JupyterLab Accessibility Statement
+
+## Edited from the [W3C accessibility statement generator](https://www.w3.org/WAI/planning/statements/generator/#create)
+
+
+## ⚠️ Draft Accessibility Statement for JupyterLab
+
+This is an accessibility statement from <span class="basic-information organization-name">Jupyter accessibility contributors</span>.
+
+### Measures to support accessibility
+
+Jupyter accessibility contributors take the following measures to ensure accessibility of JupyterLab:
+
+* Include accessibility as part of our mission statement.
+* Provide continual accessibility training for our community.
+* Assign clear accessibility goals and responsibilities.
+* Employ formal accessibility quality assurance methods.
```

Comment:
Idea: we should add something on documenting - changes, improvements and approaches to testing and QA

Reply:
These were from a list of W3C suggested ones, so I'm unclear if they map to some kind of special requirements.

I added another point.